editor/src/core/workers/parser-printer/parser-printer-arbitrary-elements.spec.ts

```diff
@@ -200,6 +206,7 @@ export var whatever = props => (
       expect.objectContaining({}),
       null,
       null,
+      addModifierExportToDetail(EmptyExportsDetail, 'whatever'),
```

Comment:
does this mean `whatever` is exported like so: `export var whatever = (props)...` ? or a separate line exporting it?

Reply:
The former.